apps/web/components/Badge.tsx

```diff
@@ -1,54 +1,72 @@
-import { proposalStatus, proposalTypes } from "@/types";
 import React from "react";
+import { proposalStatus, poolTypes } from "@/types";
+import {
+  CurrencyDollarIcon,
+  HandThumbUpIcon,
+} from "@heroicons/react/24/outline";
+import { capitalize } from "@/utils/text";
 
-interface BadgeProps {
-  type: number;
+type BadgeProps = {
+  poolType?: number;
```

Comment:
maybe its better to name it type then? because now there is this inconsistency 
![image](https://github.com/1Hive/gardens-v2/assets/13890618/b6be4f2b-a0d0-4d35-a85a-cffea96ca661)

Reply:
Ohhh I see, ok, so the poolType (funding or signaling) is taken from the proposalType data of the subgraph. Before it was named type and then I change it!. I will roll it back to type then! . Thanks!